### PR TITLE
ui/volume: rephrase the volume status from Available to Ready

### PR DESCRIPTION
**Pull request by ChengYanJin**
**Component**: ui, volume

**Context**: 
The volume status available is confusing. For example, when the PV is bound, we displayed `Available`.

**Summary**:
Rephrase the volume status from `Available` to `Ready`

**Acceptance criteria**: 
image

---

Closes: #2245

### CODE REVIEW

File: ui/src/services/NodeVolumesUtils.test.js [hunk 1]
```diff
@@ -278,7 +278,7 @@ it('should return false when volume is available and there is no PV', () => {
 
 it('should return Unkown when called with wrong input', () => {
   const volume = {
-    'status': {'conditions': []}
```
Comment (slaperche-scality) [inline ui/src/services/NodeVolumesUtils.test.js:281]: Why all those single quote changes?

Is it a linter/formatter fix? If yes, should be done in a dedicated commit, because it generates a lot of noises.

File: ui/src/services/NodeVolumesUtils.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ import {
   STATUS_AVAILABLE,
   STATUS_BOUND,
   STATUS_RELEASED,
+  STATUS_READY,
 } from '../constants';
 
 export const isVolumeDeletable = (rowData, persistentVolumes) => {
```
Comment (slaperche-scality) [inline ui/src/services/NodeVolumesUtils.js:12]: I think you forgot to replace the `STATUS_AVAILABLE` in the outer switch l.22 (for `Volume`).
Note that you should only change this one, the other one in the inner switch l.36 should stay `STATUS_AVAILABLE` (because it's about `PersistentVolume`).

File: ui/src/services/NodeVolumesUtils.js [hunk 3]
```diff
@@ -78,28 +78,28 @@ export const computeVolumeGlobalStatus = (name, status) => {
...
-  case 'False':
-    return STATUS_FAILED;
-  case 'Unknown':
-    switch(condReason) {
-    case 'Pending':
-      return STATUS_PENDING;
-    case 'Terminating':
-      return STATUS_TERMINATING;
+  switch (condStatus) {
+    case 'True':
```
Comment (slaperche-scality) [inline ui/src/services/NodeVolumesUtils.js:82]: Why did the indentation changed? If something have reformatted the code automatically, it should be done in a dedicated commit (separate functional changes from cosmetic ones).

File: ui/src/constants.js [hunk 4]
```diff
@@ -23,6 +23,7 @@ export const STATUS_FAILED = 'Failed';
 export const STATUS_AVAILABLE = 'Available';
 export const STATUS_RELEASED = 'Released';
 export const STATUS_UNKNOWN = 'Unknown';
+export const STATUS_READY = 'Ready';
```
Comment (slaperche-scality) [inline ui/src/constants.js:26]: Should explain in the commit message **WHY** we change from `Available` to `Ready` (you've have done it in the PR description, but not in the commit message).

**Review by slaperche-scality — CHANGES_REQUESTED** [review]
I think you forgot to update the code for the hint popup in `src/containers/NodeVolumes.js`: it's still using `STATUS_AVAILABLE` instead of `STATUS_READY`

File: ui/src/services/NodeVolumesUtils.test.js (hunk 1 above)
Comment (ChengYanJin) [reply ui/src/services/NodeVolumesUtils.test.js:281]: Thanks! 
The formatting has been done in a separate commit.

File: ui/src/services/NodeVolumesUtils.js [hunk 5]
```diff
@@ -79,28 +78,28 @@ export const computeVolumeGlobalStatus = (name, status) => {
...
+      return STATUS_READY;
```
Comment (slaperche-scality) [inline ui/src/services/NodeVolumesUtils.js:83]: You've also changed `AVAILABLE` to `READY` in this commit: it's not only about indentation.

I think you need to extract this part.

File: ui/src/services/NodeVolumesUtils.test.js [hunk 6]
```diff
@@ -4,6 +4,7 @@ import {
   STATUS_PENDING,
   STATUS_FAILED,
   STATUS_AVAILABLE,
```
Comment (slaperche-scality) [inline ui/src/services/NodeVolumesUtils.test.js]: I think you can remove `STATUS_AVAILABLE` from the import here: it's no longer used in this file.

File: ui/src/containers/NodeVolumes.js [hunk 7]
```diff
@@ -34,6 +34,7 @@ import {
   STATUS_FAILED,
   STATUS_AVAILABLE,
   STATUS_BOUND,
+  STATUS_READY,
```
Comment (slaperche-scality) [inline ui/src/containers/NodeVolumes.js:36]: I think you can remove `STATUS_AVAILABLE` from the import here: it's no longer used in this file.

**Review by slaperche-scality — COMMENTED** [review]
Works as expected

2 minor comments + 1 remark on a commit that wasn't correctly split.

File: ui/src/services/NodeVolumesUtils.js (hunk 5 above)
Comment (ChengYanJin) [reply ui/src/services/NodeVolumesUtils.js:83]: You are right! But I couldn't split the big switch into two hunks and commit them separately...
Let's keep it in this way then :)

**Review by slaperche-scality — DISMISSED** [review]

**Review by slaperche-scality — APPROVED** [review]
